lib/Analysis/GateAnalysis.cc

```diff
@@ -230,15 +226,23 @@ void GateAnalysisImpl::processPhi(PHINode *PN, Instruction *insertionPt) {
 
       // Or the successors unconditionally flows to an already processed block.
       // Note that there can be at most one such block.
-      for (auto &BlockValuePair : flowingValues) {
-        if (m_PDT.dominates(BlockValuePair.first, S)) {
-          SuccToVal[S] = BlockValuePair.second;
+
+      // Used to iterate over all the post-dominators of `S`.
+      BasicBlock *post_dominator = S;
```

Comment:
nit: seahorn used thisCase for local variable names.
Consider a less generic name, e.g., `postDomBlock`.